.goreleaser.yml

```diff
@@ -13,6 +14,8 @@ builds:
       - arm
       - arm64
       - mips64le
+      - 386
```

Comment:
We had removed 386 a while back. In part because of bug reports where the problem was folks downloading 386-linux and trying to run on x64 w/o 32-bit toolchain. Are distros still supporting 32-bit installs?

Reply:
@dougm I installed it on RHEL and Fedora on x86_64 and ran `govc --help` without any issues.

Reply:
@dougm Gosh, I just learned that amd64 are the ones for 'x86_64'. My intention was not to add 32-bit. So, I just updated it to only add s390x, not 386.